src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Common.Test/FilePathNormalizerTest.cs

```diff
@@ -151,8 +149,8 @@ public void Normalize_EmptyFilePath_ReturnsEmptyString()
             Assert.Equal("/", normalized);
         }
 
-        [ConditionalFact]
-        [OSSkipCondition(OperatingSystems.Windows, SkipReason = "Test only valid on non-Windows boxes")]
+
+        [OSSkipConditionFact(new[] { "WINDOWS" })]
```

Comment:
Why is Windows in Scream-Caps while Linux is written normally?

Reply:
```suggestion
        [OSSkipConditionFact(new[] { "Windows" })]
```